fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroupMgr.java

```diff
@@ -336,7 +337,12 @@ public void createWorkloadGroup(CreateWorkloadGroupStmt stmt) throws DdlExceptio
                 }
                 throw new DdlException("workload group " + workloadGroupName + " already exist");
             }
-            if (idToWorkloadGroup.size() >= Config.workload_group_max_num) {
+            if (!stmt.isInternal() && workloadGroupName.equals(StatisticConstants.STATISTICS_WORKLOAD_GROUP_NAME)) {
```

Comment:
Better use a prefix to reserve some name for internal workload group.
eg: __doris_internal_xxx
So that we can add other internal group in future easier.

Reply:
fixed